support_bundle/get_support_bundle.sh

```diff
@@ -141,7 +141,19 @@ main() {
 
     # If API key is supplied, collect streamSnap, Index settings, and fastPath settings
     if [[ ! -z ${API_KEY} ]]; then
-        API_URL=$(kubectl ${KUBE_OPTS} get cm sysdigcloud-config -o yaml | grep -i api.url: | head -1 | awk '{print$2}')
+        #Check if we are on version 6
+        VERSION_CHECK=$(kubectl ${KUBE_OPTS} get cm | grep 'sysdigcloud-api-config' | wc -l | awk '{print $1}') || true
```

Comment:
you should not need this
```
| awk '{print $1}')
```

Reply:
I agree Mark. It's been removed. I had to add this sed bit to get the output readable. It has a bunch of spaces that were throwing off setting the variable properly:

$ kubectl --namespace sysdigcloud get cm | grep 'sysdigcloud-api-config' | wc -l
       1
       
       So now I'm doing:
       kubectl --namespace sysdigcloud get cm | grep 'sysdigcloud-api-config' | wc -l | sed 's/       //g'